examples/hacker_news_assets/hacker_news_assets/repo.py

```diff
@@ -1,47 +1,23 @@
+from typing import Optional
+
 from dagster import repository, schedule_from_partitions
 
-from .assets import local_assets, prod_assets, staging_assets
-from .jobs.activity_stats import (
-    activity_stats_local_job,
-    activity_stats_prod_job,
-    activity_stats_staging_job,
-)
-from .jobs.hacker_news_api_download import (
-    download_local_job,
-    download_prod_job,
-    download_staging_job,
-)
-from .jobs.story_recommender import (
-    story_recommender_local_job,
-    story_recommender_prod_job,
-    story_recommender_staging_job,
-)
+from .assets import build_asset_group
+from .jobs.activity_stats import build_activity_stats_job
+from .jobs.hacker_news_api_download import build_download_job
+from .jobs.story_recommender import build_story_recommender_job
 from .sensors.hn_tables_updated_sensor import make_hn_tables_updated_sensor
 from .sensors.slack_on_failure_sensor import make_slack_on_failure_sensor
 
 
 @repository
-def hacker_news_assets_prod():
-    return [
-        prod_assets,
-        schedule_from_partitions(download_prod_job),
-        make_slack_on_failure_sensor(base_url="my_dagit_url.com"),
-        make_hn_tables_updated_sensor(activity_stats_prod_job),
-        make_hn_tables_updated_sensor(story_recommender_prod_job),
-    ]
-
+def hacker_news_assets(deployment_name: Optional[str]):
```

Comment:
How does this function even get the string?

Reply:
The framework would provide it. Either by grabbing a value from dagster.yaml or something equivalent in Dagster Cloud.

Reply:
So this is a projected feature, rather than some feature I don't know about.

Reply:
Yes yes correct. See PR description. Maybe should have proclaimed this more loudly in the title.